applications/MappingApplication/custom_mappers/nearest_element_mapper.h

```diff
@@ -145,6 +143,7 @@ class KRATOS_API(MAPPING_APPLICATION) NearestElementLocalSystem : public MapperL
 
     CoordinatesArrayType& Coordinates() const override
     {
+        KRATOS_DEBUG_ERROR_IF_NOT(mpNode) << "Members are not intitialized!" << std::endl;
```

Comment:
maybe also add this in the other mappers?

Reply:
Is already there, this is the exception